### PR TITLE
[bitnami/thanos] Update README

### LINKED ISSUE #6726: [bitnami/minio] WARNING: MINIO_ACCESS_KEY and MINIO_SECRET_KEY are deprecated.
**Which chart**:
minio 7.1.0

**Describe the bug**
Getting a new warning:
```
WARNING: MINIO_ACCESS_KEY and MINIO_SECRET_KEY are deprecated.
Please use MINIO_ROOT_USER and MINIO_ROOT_PASSWORD
```

**To Reproduce**

1. create a `mode: distributed` cluster 
2. watch the pod logs

**Expected behavior**
I think the templates need a update. this is also related to #4422

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
doesn't matter
```

- Output of `kubectl version`:

```
doesn't matter
```

**Additional context**
nope

### PR DESCRIPTION
**Pull request by hdavila146**
update Thanos base  configuration in readme.md, the sub chart minio change the way of auth itself with the bucket. this make crashloopbackoff to the thanos-compactor and thanos-storegateway. With this configuration is working properly already test it.

Signed-off-by: hdavila146 <87457401+hdavila146@users.noreply.github.com>

### Description of the change

The change only affect the Readme file, the minion configuration  example in thanos.

### Benefits

this new configuration works, the old  one fail for chart minion changes. 

### Possible drawbacks

none.

### Applicable issues

  - fixes #6726

### Additional information

issue https://github.com/bitnami/charts/issues/13993

### Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X ] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by hdavila146** [comment]
Hello, ready the PR sorry for the late delivery  @carrodher issue https://github.com/bitnami/charts/issues/13993

File: bitnami/thanos/README.md [hunk 1]
```diff
@@ -1340,16 +1340,16 @@ ruler:
         rules:
           - alert: "PrometheusDown"
             expr: absent(up{prometheus="monitoring/kube-prometheus"})
-metrics:
-  enabled: true
-  serviceMonitor:
-    enabled: true
```
Comment (rafariossaa) [inline bitnami/thanos/README.md:1346]: Could we get the metrics enabled or there is any issues with this ?

Comment (hdavila146) [reply bitnami/thanos/README.md:1346]: ok will do there is not problem at all i remove them for debuging.

File: bitnami/thanos/README.md [hunk 2]
```diff
@@ -1316,21 +1316,21 @@ objstoreConfig: |-
   type: s3
   config:
     bucket: thanos
-    endpoint: {{ include "thanos.minio.fullname" . }}.monitoring.svc.cluster.local:9000
+    endpoint: thanos-minio.monitoring.svc.cluster.local:9000
```
Comment (rafariossaa) [inline bitnami/thanos/README.md:1319]: This can not be harcoded, original one works well for me.

File: bitnami/thanos/README.md [hunk 3]
```diff
@@ -1316,21 +1316,21 @@ objstoreConfig: |-
...
+ compactor:
   enabled: true
-storegateway:
+ storegateway:
   enabled: true
-ruler:
+ ruler:
   enabled: true
   alertmanagers:
     - http://kube-prometheus-alertmanager.monitoring.svc.cluster.local:9093
```
Comment (rafariossaa) [inline bitnami/thanos/README.md:1336]: All this block needs to be indented, so `query` in at the same level than `config`.

**Review by rafariossaa — CHANGES_REQUESTED** [review]
Thanks for noticing this.
Could you take a look to my comments ?

File: bitnami/thanos/README.md [hunk 4]
```diff
@@ -1316,21 +1316,21 @@ objstoreConfig: |-
...
-    endpoint: {{ include "thanos.minio.fullname" . }}.monitoring.svc.cluster.local:9000
+    endpoint: thanos-minio.monitoring.svc.cluster.local:9000
     access_key: minio
     secret_key: minio123
     insecure: true
-query:
+ query:
   dnsDiscovery:
-    sidecarsService: kube-prometheus-prometheus-thanos
+    sidecarsService: bitnami-kube-prometheus-prometheus-thanos
```
Comment (pedrovalido) [inline bitnami/thanos/README.md:1325]: Why change this? the original is works well and is aligned with the values.yaml

File: bitnami/thanos/README.md [hunk 5]
```diff
@@ -1343,13 +1343,17 @@ ruler:
 metrics:
   enabled: true
   serviceMonitor:
-    enabled: true
+    enabled: true
```
Comment (rafariossaa) [inline bitnami/thanos/README.md:1346]: Remove the white space at the end of the line

File: bitnami/thanos/README.md [hunk 6]
```diff
@@ -1343,13 +1343,17 @@ ruler:
...
 minio:
-  enabled: true
   auth:
-    rootUser: "admin"
-    rootPassword: "minio123"
-  defaultBuckets: "thanos"
+    rootPassword: minio123
+    rootUser: minio
+  monitoringBuckets: thanos
+  enabled: true
```
Comment (rafariossaa) [inline bitnami/thanos/README.md:1352]: I would keep `enabled: true` as the first element in `minio:` so you can see clearly if it is enabled or not before considering the rest of parameters.

**Review by rafariossaa — CHANGES_REQUESTED** [review]
Sorry for being a little bit picky, but could you take a look to my comments ?

**Comment by rafariossaa** [comment]
Hi,
  Thanks for fixing them.
  Could you take a look to the DCO check ? https://github.com/bitnami/charts/pull/14217/checks?check_run_id=10932004943
  Also, it seem that there is a conflict in the README.md file, maybe you need to rebase your changes.

**Review by rafariossaa — APPROVED** [review]

**Pull request merged**